.github/workflows/python.yaml

```diff
@@ -39,12 +39,34 @@ jobs:
     container: python:3.8-slim
     steps:
       - uses: actions/checkout@v2
-      - name: Build
+      - name: Install dependencies
         run: |
           pip install poetry
-      - name: Release
+      - name: Build & release
         env:
           PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
         run: |
           poetry config pypi-token.pypi $PYPI_TOKEN
           poetry publish --build
+  docs:
+    needs: test
+    # if: github.ref == 'refs/heads/main'
```

Comment:
Do I understand correctly that for the desired state we would only run this on tagged releases?

Reply:
As discussed, I changed to same condition as we have for pypi deployment: if tag starts with `v`.